### PR TITLE
White spheres patch for the hero branch

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17570/HighFidelity-Beta-PR15011-50035a1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17570/HighFidelity-Beta-Interface-PR15011-50035a1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17570/HighFidelity-Beta-PR15011-50035a1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17570/HighFidelity-Beta-Interface-PR15011-50035a1.exe

**Comment by qlilaseguinolaza** [comment]
Tested PR15011 for v0.79.0.HERO-rc branch and PR Passed. No white orbs detected when banning a user and reconnecting and reloading content multiple times.

File: domain-server/src/DomainServerSettingsManager.cpp [hunk 1]
```diff
@@ -25,13 +25,13 @@
 
 #include <AccountManager.h>
 #include <Assignment.h>
+#include <AvatarData.h>
```
Comment (SimonWalton-HiFi) [inline domain-server/src/DomainServerSettingsManager.cpp]: Do we still need AvatarData.h? It's just been moved up.

Comment (Atlante45) [reply domain-server/src/DomainServerSettingsManager.cpp]: Unfortunately we do, I'd originally removed it, but it turns out that we need for another constant used underneath.

So I at least put it at its proper place (alphabetically).

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by Atlante45** [comment]
Build this please

**Comment by howard-stearns** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17642/HighFidelity-Beta-PR15011-50035a1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17642/HighFidelity-Quest-Beta-PR15011-50035a1.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17642/HighFidelity-Beta-PR15011-50035a1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17642/HighFidelity-Beta-Interface-PR15011-50035a1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17642/HighFidelity-Beta-PR15011-50035a1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17642/HighFidelity-Beta-Interface-PR15011-50035a1.exe

**Pull request merged**